apps/nextjs-website/src/_contents/ioSign/api.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiData } from '@/lib/types/apiData';
+import { ioSign } from '@/_contents/ioSign/ioSign';
+import { ioSignApiPath } from '@/_contents/ioSign/apiPath';
+
+export const ioSignApi: ApiData = {
+  ...ioSignApiPath,
+  product: ioSign,
+  specURL:
+    'https://raw.githubusercontent.com/pagopa/io-sign/main/apps/io-func-sign-issuer/openapi.yaml',
```

Comment:
```suggestion
    'https://raw.githubusercontent.com/pagopa/io-sign/latest/apps/io-func-sign-issuer/openapi.yaml',
```
we should use this link instead